build_tools/update_environments_and_lock_files.py

```diff
@@ -297,7 +297,7 @@ def remove_from(alist, to_remove):
         ],
         "package_constraints": {
             "python": "3.10",
-            "blas": "[build=mkl]",
+            "blas": "[build=openblas]",
```

Comment:
we should rename the mkl to openblas but I wanted to test without to have to change anything else.